packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -109,6 +109,7 @@ The object's type is \`TableValidityState\`, and it contains the following boole
 -   \`invalidRecordId\`: \`true\` when a record was found where \`id-field-name\` did not refer to a value of type \`string\`
 -   \`duplicateColumnId\`: \`true\` when multiple columns were defined with the same \`column-id\`
 -   \`invalidColumnId\`: \`true\` when a \`column-id\` was specified for some, but not all, columns
+-   \`invalidColumnConfiguration\`: \`true\` when one or more columns have an invalid configuration
 -   \`duplicateSortIndex\`: \`true\` when \`sort-index\` is specified as the same value for multiple columns that have \`sort-direction\` set to a value other than \`none\`
```

Comment:
It looks like we're missing `duplicateGroupIndex` here. Can you add it? It should be pretty similar to `duplicateSortIndex`. Something like:

`true` when `group-index` is specified as the same value for multiple columns